dotcom-rendering/scripts/webpack/dev/webpack.config.dev-server.js

```diff
@@ -0,0 +1,35 @@
+const chalk = require('chalk');
+
+const DEV_SERVER_PORT = 3030;
+
+console.log(
+	`${chalk.dim('DEV server running on')} ${chalk.blue.underline(
+		`http://localhost:${DEV_SERVER_PORT}`,
+	)}`,
+);
+
+module.exports = {
+	devServer: {
```

Comment:
Might be helpful, here?

```suggestion
module.exports = {
	/** @type {import('webpack-dev-server').Configuration} */
	devServer: {
```

Reply:
ts version incoming